docs/src/user/installing.md

```diff
@@ -12,30 +12,39 @@ rustup target add --toolchain stable wasm32-wasi
 
 ## Installing from Git
 
-Ambient can be installed through `cargo install`, which will download and build the repository. Our current minimum supported Rust version is <!-- rust-version-begin !-->1.67.0<!-- rust-version-end !-->, as we use recently-stabilised standard library features.
+*Ambient* can be installed through `cargo install` using git.
 
-To install the latest released version from Git, run the following:
+This will download the source and compile the runtime. Our minimum supported Rust version is <!-- rust-version-begin -->1.67.0<!-- rust-version-end -->.
+
+### Installing the latest published release
+
+This is recommended if you want a binary for your platform using the last stable release
 
 ```sh
-cargo install --git https://github.com/AmbientRun/Ambient.git --tag v0.2.1 ambient --locked
+cargo install --git https://github.com/AmbientRun/Ambient.git --tag v0.2.1 --locked --force ambient
 ```
 
-To install the latest version on the `main` branch, run the following:
+**Note**: If you are running a project outside of the `guest/rust` workspace, it is likely that the published version of the API will be incompatible with `main`, and you will need to specify the dependency manually.
```

Comment:
Move this back to latest development version (it's not relevant to the latest published release)